src/actions/guides/deploying/ubuntu.cr

```diff
@@ -55,7 +55,7 @@ class Guides::Deploying::Ubuntu < GuideAction
 
     ```bash
     sudo mkdir /srv/<yourapp>
-    chown deploy:deploy /srv/<yourapp>
+    sudo chown deploy:deploy /srv/<yourapp>
```

Comment:
does this also need to be `sudo chown -R deploy:deploy /srv/<yourapp>`? Or does it not matter if it's recursive?

Reply:
No it doesn't have to be recursive as the directory is supposed to be empty: it was created with the `mkdir` just above.

Reply:
oh right 😅  still too early here. I wasn't finished with ☕  yet 😛